From 1f1e1db3e9a0a445663af6cbc9fbac4eb9e73d05 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Thu, 4 Apr 2019 16:42:29 +0800
Subject: [PATCH 05/30] drm/imx/dcss: use drm_property_blob_put instead of
 drm_property_unreference_blob

In upstream commit 3ae7fb202d86 ("drm: Remove drm_property_{un/reference}_blob aliases"),
the function "drm_property_unreference_blob" has been deleted and use drm_property_blob_put
instead.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/gpu/drm/imx/dcss/dcss-crtc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/dcss/dcss-crtc.c b/drivers/gpu/drm/imx/dcss/dcss-crtc.c
index 9cf8ef64d731..47303b4ccebb 100644
--- a/drivers/gpu/drm/imx/dcss/dcss-crtc.c
+++ b/drivers/gpu/drm/imx/dcss/dcss-crtc.c
@@ -58,7 +58,7 @@ static void dcss_crtc_reset(struct drm_crtc *crtc)
 
 	if (crtc->state) {
 		if (crtc->state->mode_blob)
-			drm_property_unreference_blob(crtc->state->mode_blob);
+			drm_property_blob_put(crtc->state->mode_blob);
 
 		state = to_imx_crtc_state(crtc->state);
 		memset(state, 0, sizeof(*state));
-- 
2.17.1

